.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release-unix:
+    name: Release for ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: macos-15-intel
+            name: x86_64-macos
+          - os: macos-15
+            name: aarch64-macos
+          - os: ubuntu-latest
+            name: x86_64-linux
+          - os: ubuntu-latest
+            name: x86_64-linux-statically-linked
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - uses: ocaml-dune/setup-dune@v0
+
+      - name: Load dependencies from the cache
+        id: artifact-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            _build
+            ~/.cache/dune
+          key: ${{ matrix.os }}-artifacts-${{ hashFiles('dune.lock') }}
+
+      - name: Patch bin/dune to statically link executable
+        if: matrix.name == 'x86_64-linux-statically-linked'
+        run: |
+          awk '{ print } /\(executable/ { print " (link_flags (:standard -cclib -static))" }' bin/dune > bin/dune.static
```

Comment:
Yowch :-) Can't this be integrated as a profile or something?

Reply:
Possibly! I'll see what I can come up with.

Reply:
Ok yes this can absolutely be done with profiles, simply by appending the following to bin/dune:
```
(env
 (static
  (link_flags
   (:standard -cclib -static))))
```

I've updated the PR to include this and to just set `DUNE_PROFILE=static`.